src/main-process/atom-window.js

```diff
@@ -48,7 +48,8 @@ module.exports = class AtomWindow extends EventEmitter {
         // response to a middle-click.
         // (Ref: https://github.com/atom/atom/pull/12696#issuecomment-290496960)
         disableBlinkFeatures: 'Auxclick',
-        nodeIntegration: true
+        nodeIntegration: true,
+        webview: true
```

Comment:
Should this be `webviewTag : true`? 

I seen in the electron webview documentation:
```
webviewTag Boolean (optional) - Whether to enable the <webview> tag. Defaults to false. Note: The preload script configured for the <webview> will have node integration enabled when it is executed so you should ensure remote/untrusted content is not able to create a <webview> tag with a possibly malicious preload script. You can use the will-attach-webview event on webContents to strip away the preload script and to validate or alter the <webview>'s initial settings.
```

Reply:
Yes, it was an oversight, corrected by this PR https://github.com/atom/atom/pull/20855